hack/start-build.sh

```diff
@@ -16,12 +16,6 @@ then
         "containers": [
           {
             "name": "ingress-operator",
-            "env": [
-              {
-                "name": "IMAGE",
-                "value": "image-registry.openshift-image-registry.svc:5000/openshift-ingress-operator/ingress-operator:latest"
-              }
-            ],
```

Comment:
Don't we need to keep 'IMAGE' env variable to update haproxy version?
I think we may need IMAGE env but the value should point to haproxy instead of ingress-operator image.

Reply:
The buildconfig only rebuilds the operator image, so `start-build.sh` should leave the existing `IMAGE` variable alone.